helloworld/test/goodbye-test.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const expect = require('chai').expect;
+const goodbye = require('../lib/goodbye.js');
+
+describe('testing module goodbye.js', function(){
+  describe('testing method cya', function(){
+    it('should return "see ya later bangarang"', function(){
+      let result = goodbye.cya('bangarang');
+      expect(result).to.equal('sea ya later bangarang');
+    });
+  });
+
+  describe('testing method tatatu', function(){
```

Comment:
See my above comment on not needing to include the ```goodbye``` modules. This test failed because of a spelling mismatch, but I don't think it was a test you wrote, just example code. 

In the future, Duncan will include an extra process it so that all your tests **must** pass before I can even grade your work. If tests aren't passing, I can't review it and it's an automatic 0. So just an FYI of why it might be good to .gitignore this stuff if you're just using it for reference. :)